Makefile

```diff
@@ -0,0 +1,10 @@
+NAME = stream-cli
+
+$(NAME):
+	@go install ./cmd/$(NAME)
+
+build:
+	@go build ./cmd/$(NAME)
+
+test:
+	@go test -v ./...

```

Comment:
let's have goodies from backend makefiles to prevent common issues

Reply:
which kind of common issues?

Reply:
shell, error on pipes, warning for undefined, etc.